composer.json

```diff
@@ -25,6 +25,16 @@
         "scripts/govcms-lint-distro",
         "scripts/govcms-phpunit",
         "scripts/govcms-vet",
-        "scripts/govcms-deploy"
```

Comment:
if we leave govcms-deploy in there for a transition period, we can get this merged faster, no?

Reply:
Quite, should deprecate over backwards breaking changes :D